To: vim_dev@googlegroups.com
Subject: Patch 7.3.229
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.229
Problem:    Using fork() makes gvim crash on Mac when build with
            CoreFoundation.
Solution:   Disallow fork() when __APPLE__ is defined. (Hisashi T Fujinaka)
Files:      src/gui.c


*** ../mercurial/vim73/src/gui.c	2011-05-10 16:41:13.000000000 +0200
--- src/gui.c	2011-06-20 00:51:21.000000000 +0200
***************
*** 59,65 ****
  gui_start()
  {
      char_u	*old_term;
! #if defined(UNIX) && !defined(__BEOS__) && !defined(MACOS_X)
  # define MAY_FORK
      int		dofork = TRUE;
  #endif
--- 59,66 ----
  gui_start()
  {
      char_u	*old_term;
! #if defined(UNIX) && !defined(__BEOS__) && !defined(MACOS_X) \
! 	&& !defined(__APPLE__)
  # define MAY_FORK
      int		dofork = TRUE;
  #endif
*** ../vim-7.3.228/src/version.c	2011-06-20 00:45:55.000000000 +0200
--- src/version.c	2011-06-20 00:50:42.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     229,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
198. You read all the quotes at Netaholics Anonymous and keep thinking
     "What's wrong with that?"

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
